tictactoe.iml

```diff
@@ -11,5 +11,7 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Maven: junit:junit:4.12" level="project" />
```

Comment:
You should let Maven handle dependency resolution. I think IntelliJ has functionality to "import" Maven projects. Are you using that? That would let you make updates to your pom.xml and IntelliJ will know where to find everything it needs to find.

Reply:
I opted-in to auto-import of maven projects - I'm not sure if that is the same thing you are referring to. After I opted-in nothing was changed in my .iml file however.

Reply:
Ok. If you are using auto-import then that's perfect! If you change the dependencies in your pom.xml IntelliJ might update this file to keep it in sync. I'm not too sure what IntelliJ does when it "auto-imports" stuff.

Nothing to change, then.